packages/main/src/CardHeader.js

```diff
@@ -0,0 +1,302 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { getEffectiveAriaLabelText } from "@ui5/webcomponents-base/dist/util/AriaLabelHelper.js";
+import { isSpace, isEnter } from "@ui5/webcomponents-base/dist/Keys.js";
+import CardHeaderTemplate from "./generated/templates/CardHeaderTemplate.lit.js";
+import Icon from "./Icon.js";
+
+import {
+	AVATAR_TOOLTIP,
+	ARIA_ROLEDESCRIPTION_CARD_HEADER,
+	ARIA_ROLEDESCRIPTION_INTERACTIVE_CARD_HEADER,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import cardHeaderCss from "./generated/themes/CardHeader.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-card-header",
+	languageAware: true,
+	managedSlots: true,
+	slots: /** @lends sap.ui.webcomponents.main.CardHeader.prototype */ {
+
+		/**
+		 * Defines an avatar image, displayed in the left most part of the header.
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		avatar: {
+			type: HTMLElement,
+		},
+
+		/**
+		 * Defines an action, displayed in the right most part of the header.
+		 * <br><br>
+		 * <b>Note:</b> If set, the <code>status</code> text will not be displayed,
+		 * you can either have <code>action</code>, or <code>status</code>.
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @public
+		 */
+		action: {
+			type: HTMLElement,
+		},
+	},
+	properties: /** @lends sap.ui.webcomponents.main.CardHeader.prototype */ {
+
+		/**
+		 * Defines the title displayed in the component header.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		titleText: {
+			type: String,
+		},
+
+		/**
+		 * Defines the subtitle displayed in the component header.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		subtitleText: {
+			type: String,
+		},
+
+		/**
+		 * Defines the status displayed in the component header.
+		 * <br><br>
+		 * <b>Note:</b> If the <code>action</code> slot is set, the <code>status</code> will not be displayed,
+		 * you can either have <code>action</code>, or <code>status</code>.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		status: {
+			type: String,
+		},
+
+		/**
+		 * Defines if the component header would be interactive,
+		 * e.g gets hover effect, gets focused and <code>headerPress</code> event is fired, when it is pressed.
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		headerInteractive: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines the aria-label attribute for the component
+		 *
+		 * @type {String}
+		 * @private
+		 * @defaultvalue ""
+		 */
+		ariaLabel: {
```

Comment:
There is one thing that has to be handled later on.

There is [change](https://github.com/SAP/ui5-webcomponents/pull/3449), related to the a11y of all components, including the Card, that has been merged. Basically, all "**ariaLabel**" props have been replaced by public "**accessibleName**" prop.

Here is the line, changing it in the Card.js and this is in the master:
https://github.com/SAP/ui5-webcomponents/pull/3449/files#diff-68553941f66b78ef4c1a908ddbee4a9d9a9f0f89f6c3532bc2f697c56bae61cbL118

However, I know from @elenastoyanovaa  and @fifoosid that there is a discussion to remove the **accessibleName** (and accessibleNameRef, which the replacement of the ariaLabelledby) from the Card.
So, we either remove both **ariaLabel** and **ariaLabelledby**, or replace them with **accessibleName** and **accessibleNameRef**

@fifoosid and @elenastoyanovaa could you elaborate on this, so we decide how to proceed with this.

Reply:
After I talked with Filip, the decision is to remove **ariaLabel** and **ariaLabelledby**. 
But, we will do this clean up afterwards.